### PR TITLE
Add role argument spec and action to auto-update readme

### PR DESCRIPTION
**Pull request by rndmh3ro**
This adds a role argument spec to the role so it can be checked at runtime if all required parameters are set.
This helps us prevent issues like https://github.com/telekom-mms/ansible-collection-icinga-director/issues/34

### CODE REVIEW

File: changelogs/changelog.yaml [hunk 1]
```diff
@@ -285,4 +285,6 @@ releases:
         - Update test matrix (https://github.com/telekom-mms/ansible-collection-icinga-director/pull/223)
         - exclude old python versions with ansible devel (https://github.com/telekom-mms/ansible-collection-icinga-director/pull/225)
         - Update actions/checkout action to v4 (https://github.com/telekom-mms/ansible-collection-icinga-director/pull/224)
+        - use telekom-mms renovate preset (https://github.com/telekom-mms/ansible-collection-icinga-director/pull/229)
```
[the remark this answers is not in the document]
Comment (rndmh3ro) [reply changelogs/changelog.yaml:288]: Yeah, they do not belong here, thanks for noticing.

File: roles/ansible_icinga/tasks/icinga_command.yml [hunk 2]
```diff
@@ -11,21 +9,21 @@
...
+    state: "{{ command.state | default(omit) }}"
+    command: "{{ command.command | default(omit) }}"
+    command_type: "{{ command.command_type | default(icinga_command_type) }}"
+    disabled: "{{ command.disabled | default(icinga_command_disabled) }}"
+    object_name: "{{ command.name }}"
+    imports: "{{ command.imports | default(omit) }}"
+    timeout: "{{ command.timeout | default(omit) }}"
+    zone: "{{ command.zone | default(omit) }}"
+    vars: "{{ command.vars | default(omit) }}"
+    arguments: "{{ command.arguments | default(omit) }}"
```
[the remark this answers is not in the document]
Comment (rndmh3ro) [reply roles/ansible_icinga/tasks/icinga_command.yml:21]: I just tried it and it didn't work...

Anyway for me that's too much magic. :)

File: .aar_doc.yml [hunk 3]
```diff
@@ -0,0 +1,72 @@
+output_template: |
```
[the remark this answers is not in the document]
Comment (rndmh3ro) [reply .aar_doc.yml:1]: Probably - I'll remove it.

File: roles/ansible_icinga/meta/argument_specs.yml [hunk 4]
```diff
@@ -0,0 +1,1345 @@
...
+            description:
+              - Whether the agent is configured to accept config.
+            type: bool
+            choices:
+              - true
+              - false
+            version_added: 1.9.0
+      icinga_command_templates:
+        description:
+          - A list of Icinga command_templat to configure
```
Comment (schurzi) [inline roles/ansible_icinga/meta/argument_specs.yml:543]:
```suggestion
          - A list of Icinga command_templates to configure
```

File: roles/ansible_icinga/meta/argument_specs.yml [hunk 5]
```diff
@@ -0,0 +1,1345 @@
...
+            description:
+              - Disabled objects will not be deployed.
+            type: bool
+            default: false
+            choices:
+              - true
+              - false
+      icinga_service_templates:
+        description:
+          - A list of Icinga service_templat to configure
```
Comment (schurzi) [inline roles/ansible_icinga/meta/argument_specs.yml:865]:
```suggestion
          - A list of Icinga service_templates to configure
```

**Review by schurzi — APPROVED** [review]

**Pull request merged**